docs/tables/aws_kinesis_consumer.md

```diff
@@ -0,0 +1,74 @@
+# Table: aws_kinesis_consumer
+
+A consumer, known as an Amazon Kinesis Data Streams application, is an application that you build to read and process data records from Kinesis data streams.
+
+## Examples
+
+### List all consumers
+
+```sql
+select
+  consumer_name,
+  consumer_arn,
+  consumer_status,
+  stream_arn
+from
+  aws_kinesis_consumer;
+```
+
+
+### Get a specific consumers
```

Comment:
Get a specific consumers > Should be consumer we can write `Get details of specific consumer`